tests/integration/auth/test_auth.py

```diff
@@ -0,0 +1,18 @@
+import pytest  # type: ignore
+from nisystemlink.clients.auth import AuthClient
+
+
+@pytest.fixture(scope="class")
+def client(enterprise_config):
+    """Fixture to create a AuthClient instance."""
+    return AuthClient(enterprise_config)
+
+
+@pytest.mark.enterprise
+@pytest.mark.integration
+class TestAuth:
+    def test__get_auth__returns(self, client: AuthClient):
+        auth_response = client.get_auth()
+        assert getattr(auth_response, "error") is None
+        assert auth_response.workspaces is not None
+        assert len(auth_response.workspaces) >= 1
```

Comment:
It would be better if this validated more of the fields in the response. Although it will admittedly be a bit of work to set up. We'd need the test to:
- Create a new user
- Generate an API key for that user
- Create an auth client with the new configuration
- Then call `.get_auth()`and assert the fields of the created user

I think this is something we should do, but I'm okay with having it in a separate PR that follows this one.